tests/unit/meta/conftest.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+import pytest
+from click.testing import CliRunner
+
+
+@pytest.fixture
+def runner() -> CliRunner:
+    """Fixture for creating a CliRunner instance."""
+    return CliRunner()
+
+
+@pytest.fixture
+def program() -> str:
+    return "cbds"
+
+
+@pytest.fixture
+def project() -> str:
+    project = uuid.uuid4().hex.replace('-', '_')
+    return project
+
+
+@pytest.fixture
+def project_id(program, project) -> str:
+    project_id = f"{program}-{project}"
+    return project_id
```

Comment:
consolidate into `tests/conftest.py` rather than have it in both `test/integration` and `tests/unit`?

Reply:
keeping it separate for now